tests/Microsoft.Identity.Test.Unit/pop/PoPTests.cs

```diff
@@ -154,11 +156,15 @@ public async Task POP_WithMissingNonceForPCA_Async()
         {
             using (var httpManager = new MockHttpManager())
             {
-                PublicClientApplication app =
-                    PublicClientApplicationBuilder.Create(TestConstants.ClientId)
-                                                              .WithBrokerPreview()
-                                                              .WithHttpManager(httpManager)
-                                                              .BuildConcrete();
+                
+                var pcaBuilder = PublicClientApplicationBuilder.Create(TestConstants.ClientId)
+                                                              .WithHttpManager(httpManager);
+#if NET6_WIN
+                pcaBuilder = pcaBuilder.WithBroker(true);
+#else
+                pcaBuilder = Client.Broker.BrokerExtension.WithWindowsBroker(pcaBuilder);
```

Comment:
I would invoke the method via the builder, since that's how it will be used and it still tests the extension method.

Reply:
Done.